utils.py

```diff
@@ -111,30 +116,78 @@ def make_confusion_matrix(y_true, y_pred, classes=None, figsize=(10, 10), text_s
                 size=text_size,
             )
     plt.show()
+    # path = str()
+    # for name in classes:
+    #     path += name.replace(' ', '-').lower()
+    # path += '.png'
+    # plt.savefig(f"plots/{path}")
 
 
 def test_model(model: nn.Module, test_data: DataLoader):
     """
-    Test the performance of the trained ComposerClassifier model with the provided test data loader
-    .by plotting a confusion matrix.
+    Test the performance of the trained ComposerClassifier model with the provided test data loader.
 
     Args:
         model (PitchSeqNN): The trained ComposerClassifier model to be evaluated.
         test_data (DataLoader): Data for the model to be tested on
     Returns:
-        Tupre[int, int]: Tuple of (true_labels, predicted_labels)
+        dict: all sample data updated with "pred" key.
     """
     # containers for predictions and truths:
     predicted = torch.tensor([])
     true = torch.tensor([])
+    data = {}
     with torch.no_grad():
-        for data in test_data:
-            notes, labels = data
-            out = model(notes)
+        for samples in test_data:
```

Comment:
```suggestion
        for batch in test_data:
```

It's good to be consistent with the variable names, and when iterating over a `DataLoader`, we get batches